lib/smart_proxy_remote_execution_ssh/multiplexed_ssh_connection.rb

```diff
@@ -60,6 +60,10 @@ def initialize(options, logger:)
       @host_public_key = options.fetch(:host_public_key, nil)
       @verify_host = options.fetch(:verify_host, nil)
       @client_private_key_file = settings.ssh_identity_key_file
+      @client_ca_known_hosts_file = settings.ssh_ca_known_hosts_file
+      @client_ca_public_key_file = settings.ssh_user_ca_public_key_file
+      @client_ca_cert_file = settings.ssh_ca_cert_file
+      @client_cert_file = Proxy::RemoteExecution::Ssh.cert_file
```

Comment:
Why do you treat these inconsistently? There is also `Proxy::RemoteExecution::Ssh.ca_public_key_file` but you don't use it. And for the others, there is no such method.

Reply:
It relates to my question to the comment above: https://github.com/theforeman/smart_proxy_remote_execution_ssh/pull/126#discussion_r2199775536

cert_file is, for now,  a non-editable value whose name depends on the identity key file, so it is not in settings. Personally, I do not see any problem to making the cert file name an editable value, I just wanted to confirm first whether there are any objections against it.